addons/storysource/loader.js

```diff
@@ -1 +1 @@
-module.exports = require('./dist/loader');
+module.exports = require('@storybook/source-loader');
```

Comment:
for backward compatibility, but it will be recommended to change the usage in webpack of course.